restructuredtext_lint/cli.py

```diff
@@ -5,20 +5,36 @@
 import os
 import sys
 
+from collections import OrderedDict
+
+from docutils.utils import Reporter
+
 from restructuredtext_lint.lint import lint_file
 
+# Generate our levels mapping for humans, using ordered dict for --help string
+# http://repo.or.cz/docutils.git/blob/422cede485668203abc01c76ca317578ff634b30:/docutils/docutils/utils/__init__.py#l65
+LEVEL_MAP = OrderedDict([
```

Comment:
Ah, nice. I typically don't read `git commit` content as it breaks development flow. I now see the comment 2 lines above -- sorry about that x_x